src/Core/WinSWCore/ServiceDescriptor.cs

```diff
@@ -698,5 +698,7 @@ public ProcessPriorityClass Priority
                 return (ProcessPriorityClass)Enum.Parse(typeof(ProcessPriorityClass), p, true);
             }
         }
+
+        public string? SecurityDescriptor => SingleElement("securityDescriptor", true);

```

Comment:
reference the required format in the documentation?
https://docs.microsoft.com/en-us/windows/win32/secauthz/security-descriptor-definition-language